projects/lib/package.json

```diff
@@ -82,5 +79,8 @@
                 }
             }
         }
+    },
+    "dependencies": {
+        "@finos/fdc3": "^2.2.0-alpha.3"
```

Comment:
this should probably be a peer actually (I realise that it was not a peer already).